tests/layers/test_signal.py

```diff
@@ -241,3 +241,50 @@ def test_mel_spectrogram(
     )(x).numpy()
     # These tolerances make look large, but the values we're comparing are in the 10e2-10e4 range.
     assert np.allclose(librosa_spec, rgp_spec, atol=1e-2, rtol=0)
+
+def test_magnitude() -> None:
```

Comment:
Would it be possible to add a few other test cases using `pytest.mark.parametrize`?

Just a few obvious cases, e.g., `[1], [-1], [0], [1,1]`.

Reply:
Add test cases:
* Random
* [[0],[1],[2],[1j],[2j],],
* [[1+2j],[3-4j],[-5+6j],[-7-8j],[9+10j],],